blocks/commerce-account-sidebar/commerce-account-sidebar.js

```diff
@@ -1,7 +1,7 @@
 import { Icon, provider as uiProvider } from '@dropins/tools/components.js';
 import { render as accountRenderer } from '@dropins/storefront-account/render.js';
 import { loadFragment } from '../fragment/fragment.js';
-import { CUSTOMER_ORDERS_PATH } from '../../scripts/constants.js';
```

Comment:
I don't know how I feel about constants moving to commerce. Do you think they "belong" to commerce? They definitely are in this grey area.